tests/test_span_readable.py

```diff
@@ -0,0 +1,65 @@
+import time
+from instana.span.readable_span import Event, ReadableSpan
+from instana.span_context import SpanContext
+from opentelemetry.trace.status import Status, StatusCode
+
+from instana.w3c_trace_context.traceparent import Traceparent
+
+
+def test_readablespan(
+    span_context: SpanContext,
+    trace_id: int,
+    span_id: int,
+) -> None:
+    span_name = "test-span"
+    timestamp = time.time_ns()
+    span = ReadableSpan(span_name, span_context)
+
+    assert span is not None
+    assert isinstance(span, ReadableSpan)
+    assert span.name == span_name
+
+    context = span.context
```

Comment:
```suggestion
    span_context = span.get_span_context()
```

Just a small suggestion. Since `Context` object is different from a `SpanContext` object, I would suggest renaming the variable and explicitly calling the `span.get_span_context()` method instead of directly accessing the `span.context` attribute. This is just for clarity, the result will be the same as earlier.

Reply:
+1 on Varsha's suggestion.

Reply:
ReadableSpan doesn't have get_span_context function

Reply:
@CagriYonca, so the `get_span_context` method must be moved from `span.py` to the `readable_span.py` file.
Can you make that change as well (in a specific commit)?

Reply:
Done!